modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua

```diff
@@ -60,41 +21,6 @@ function index()
 
 	-- backend
 	assign({"mini", "freifunk"}, {"admin", "freifunk"}, _("Freifunk"), 5)
```

Comment:
@jow- is there something equivalent to assign in the new json-based menu?

Reply:
@andibraeu - theoretically you just need to add different menu items invoking the same action (view, template, call, etc.)

Reply:
But it is not possible to "inherit" a complete submenu structure from another package like in https://github.com/freifunk/openwrt-packages/pull/54/files/0e774e74610af494e083955b2b7b4c202419c514#diff-190ca556edab3bbfceec524026162027986e8205e5322a20612882b22829bda7L52?